skops/io/_general.py

```diff
@@ -532,6 +533,37 @@ def _construct(self):
         return content_bytearray
 
 
+def operator_func_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
+    _, attrs = obj.__reduce__()
+    res = {
+        "__class__": obj.__class__.__name__,
+        "__module__": "operator",
+        "__loader__": "OperatorFuncNode",
+        "attrs": attrs,
+    }
+    return res
+
+
+class OperatorFuncNode(Node):
+    def __init__(
+        self,
+        state: dict[str, Any],
+        load_context: LoadContext,
+        trusted: bool | Sequence[str] = False,
+    ) -> None:
+        super().__init__(state, load_context, trusted)
+        self.trusted = self._get_trusted(trusted, [])
+        self.children["attrs"] = state["attrs"]
+
+    def _construct(self):
+        op = getattr(operator, self.class_name)
+        return op(*self.children["attrs"])
+
+
+# <class 'builtin_function_or_method'>
+builtin_function_or_method = type(len)
```

Comment:
what made you add this? Is this how we support `neg` for example? Can't be.

Reply:
Some operators can reasonably be used without instantiating when they don't take an argument, e.g. `operator.neg`. Therefore, we have to match their type, which is `<class 'builtin_function_or_method'>`. I couldn't find a nice way to reference that type, which is why I used `type(len)`, but it doesn't really have to be `len`, could be something else. If you know a better method, please let me know.